gnd/src/main/java/com/google/android/gnd/MainViewModel.java

```diff
@@ -50,6 +50,8 @@ public MainViewModel(
     this.navigator = navigator;
 
     // TODO: Move to background service.
+    // Add the project repository subscription to the collection of subscriptions which will be
+    // disposed when the view model is cleared
```

Comment:
Should this be explained in the declaration `disposeOnClear()` instead? It's used widely throughout.

Reply:
The confusion for me was that I'm in the constructor and the first method talks about disposing something. I'm thinking "why are we talking about disposing objects when we're creating one?". Digging further it makes sense: the subscription (a `Disposable`) is added to a collection of subscriptions which should be unsubscribed and disposed of when the view model is cleared. 

Perhaps it might read better if split onto 2 lines. Have refactored. WDYT?